hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -315,6 +315,17 @@ public void rollbackFailedBootstrap() {
    */
   public abstract O insert(I records, final String instantTime);
 
+  /**
+   * Inserts the given Error HoodieRecords, into the table. This API is intended to be used for normal writes.
+   * <p>
+   * This API is intended to be used to write wrong records.
+   *
+   * @param records Error HoodieRecords to insert
+   * @param instantTime Instant time of the commit
+   * @return Collection of WriteStatus to inspect errors and counts
+   */
+  public abstract O insertError(I records, final String instantTime);
```

Comment:
So this api inserts error records into a separate error table or the regular Hudi table? The java doc seems to be misleading. Can we mention into the error table?

Reply:
sure，done